include/mgp.hpp

```diff
@@ -3185,6 +3226,44 @@ inline bool Value::IsDuration() const { return mgp::value_is_duration(ptr_); }
 inline bool Value::operator==(const Value &other) const { return util::ValuesEqual(ptr_, other.ptr_); }
 
 inline bool Value::operator!=(const Value &other) const { return !(*this == other); }
+
+inline const std::string Value::TypeString() const {
+  const mgp::Type &valueType = Type();
+
+  switch (valueType) {
+    case Type::Null:
+      return "Type: Null";
```

Comment:
Maybe it makes sense to just return Null, Node, etc.. so user can decide on what kind of error message will be written after that

Reply:
So instead of Type::Null, return just Null, etc. Does that make more sense?

Reply:
Yeah, will do